### ISSUE TITLE
VIPER: Uncaught exception in metrics retrieval

### ISSUE THREAD

**Issue by pkraker** [OpenKnowledgeMaps/Headstart#244]
For certain projects (see e.g. EC/647557), `enrich_output_json` encounters an uncaught exception, thus preventing the creation of a map for these projects.

Error:
```
Error in doc_parse_raw(x, encoding = encoding, base_url = base_url, as_html = as_html,  : 
  Opening and ending tag mismatch: link line 5 and head [76]
```

**Comment by chreman** [comment]
at some point, malformed DOIs enter the dataset, e.g. "10.1007/s00778-016-0424-7\n\t\t\t", the proposed fix #245 should remove those cases.

**Comment by pkraker** [comment]
Thanks, the PR does fix the issue.
